src/lamp_py/ingestion/glides.py

```diff
@@ -98,6 +99,8 @@ def append_records(self) -> None:
         now = datetime.now()
         start = datetime(2024, 1, 1)
 
+        unique_ts_min = now - timedelta(hours=36)
+
```

Comment:
used for uniquing the datasets. the kinesis stream holds data for 24 hours, so we'll run the unique on any data older than 36 hours to be safe.